JavaScriptSPA/index.html

```diff
@@ -19,8 +19,9 @@
 <script>
     var msalConfig = {
         auth: {
-            clientId: 'Enter_the_Application_Id_here', //This is your client ID
-            authority: "https://login.microsoftonline.com/Enter_the_Tenant_Info_Here" //This is your tenant info
+            clientId: 'Enter_the_Application_Id_here', // This is your client ID
+            authority: "https://login.microsoftonline.com/Enter_the_Tenant_Info_Here", // This is your tenant info
+            redirectUri: "http://localhost:30662" // to set the reply URL as process.env.PORT, configured in server.js
```

Comment:
Maybe we can make this generic. Maybe we can add a comment. This is just a follow-up OR for the issues raised by the CXP.

Reply:
maybe add the comment, that this should be the whitelisted url set in their aad application and that if it is not set, it will default to the page they make the call on ?